coretex/utils/process.py

```diff
@@ -40,37 +40,53 @@ class CommandException(Exception):
     pass
 
 
-def command(args: List[str], ignoreOutput: bool = False, shell: bool = False) -> None:
+def command(
+        args: List[str],
+        ignoreStdout: bool = False,
+        ignoreStderr: bool = False,
+        shell: bool = False,
+        check: bool = True
+    ) -> Tuple[int, str, str]:
+
     process = subprocess.Popen(
         args,
         shell = shell,
         cwd = Path(__file__).parent,
-        stdout = None if ignoreOutput else subprocess.PIPE,
-        stderr = None if ignoreOutput else subprocess.PIPE
+        stdout = subprocess.PIPE,
+        stderr = subprocess.PIPE
     )
 
+    stdOutStr: str = ''
+    stdErrStr: str = ''
```

Comment:
No need for explicit type annotation, also use double quotes instead of single ones.